### ISSUE TITLE
Incorrect compose compiler version mentionned in beta release

### ISSUE THREAD

**Issue by codlab** [JetBrains/compose-multiplatform#3735]
**Describe the bug**
Incorrect compose compiler version mentionned in beta release

**Versions**
- Kotlin version*: 1.5.10-beta02
- Compose Multiplatform version*: 1.9.20-beta02

The commit https://github.com/JetBrains/compose-multiplatform/commit/393cfdd6638eee465b3c974fe9e6b3b0a1db57c1 shows that the ComposeCompiler is referring to 1.5.2.1-Beta2 and 1.5.2.1-Beta3. However the pre-release template shows that the corresponding expected version for the compose compiler is 1.5.3. 

I understand that it seems the goal is for the compose compiler version to be changed later to 1.5.3 but this is misleading. A 2.5.3-betaX should either be released or the pre-release statement should highlight the expected "temporary" version

**Comment by eymar** [comment]
Where did you see '2.5.3' version? There is no such version afaik.

This commit https://github.com/JetBrains/compose-multiplatform/commit/393cfdd6638eee465b3c974fe9e6b3b0a1db57c1 has this line:
 "1.9.20-Beta2" to "1.5.2.1-Beta3" 
 
 and it's correct. 
 Beta2 -> Beta3 is not a mistake since kotlin and compose compiler plugin are indepdenent things with their own versioning.

**Comment by eymar** [comment]
Or do you refer to the Dependencies section at this page https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.5.10-beta02?
It mentions [Compiler 1.5.3](https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.3) as a dependency.

1.5.2.1-Beta3 is based on Jetpack Compose [Compiler 1.5.3](https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.3)

There is note in that section:
> This version of Compose Multiplatform is based on the next Jetpack Compose libraries:

So "Dependencies" there is a list of Jetpack Compose libs on top of which Compose Multiplatform release is built.

**Comment by codlab** [comment]
My bad, I just edited the ticket with the proper 1.5. instead of 2.5.

I was testing the 1.9.20-Beta01 right before, so bumped to Beta02 and I reverted the bump to this beta version due to a message telling that the version of the compiler I was using (1.5.3) was incorrect and incompatible which lead to investigate about it and stumbled into this commit showing the compiler version used was 1.5.2.1-Beta3. However, I just tried back again after a full cleanup, this disappeared.

I'll close this ticket
